dwave/plugins/torch/boltzmann_machine.py

```diff
@@ -89,6 +105,28 @@ def __init__(
         #     defined above
         self._setup_hidden()
 
+    def set_linear(self, linear: dict[tuple[Hashable], float]) -> None:
+        """Set linear biases of the model.
+
+        Args:
+            linear (dict[tuple[Hashable], float]): A dictionary mapping from nodes of the model to
+                its corresponding linear bias.
```

Comment:
Similarly here and for `set_quadratic`, could clarify that not all nodes need to be set, and the rest will be unchanged.